exercises/kindergarten-garden/test/Tests.hs

```diff
@@ -47,18 +47,10 @@ specs = describe "kindergarten-garden" $ do
 
       plants "Alice"   `shouldBe` [ Violets , Radishes, Violets , Radishes ]
       plants "Bob"     `shouldBe` [ Clover  , Grass   , Clover  , Clover   ]
-      plants "Charlie" `shouldBe` [ Violets , Violets , Clover  , Grass    ]
```

Comment:
we could keep these since we already have them, and just declare that they're Haskell specific. don't really care (but note that this is the only place that many of these names are tested!)